client/src/components/App.js

```diff
@@ -59,13 +38,9 @@ export default class App extends Component {
       <div className="App">
         {!userSession.isUserSignedIn() ?
           <Login handleSignIn={this.handleSignIn} />
-          :
-          <div>
-            <NavBar />
+          : <div>
             <Button onClick={this.handleSignOut}>Sign Out</Button>
-            <div>
-              <p>Welcome, { userSession.loadUserData().profile.name}</p>
-            </div>
+            <NavBar />
           </div>
         }
       </div>
```

Comment:
Did you intend to edit up `App.js` here? If not, could you rebase off master if possible in order to make sure you're only touching the scope of this PR?

Reply:
I used App.js to render the charts to see how they looks like. I see that I change order of the DOM elements compared to the master.
It's been rebased.

Reply:
Hrm, not sure the issue as I'm not super familiar with working with forked branches, but it looks like there is a lot more changed here - seems like you're still on an older version of master (if you compare current master to your master, you'll see there's no handling of user data in your `App.js` at least according to this PR). Let me know if anything needs clearing up!

Reply:
Looking good now, thanks!